src/cloud-api-adaptor/podvm/files/etc/agent-config.toml

```diff
@@ -7,7 +7,10 @@ server_addr="unix:///run/kata-containers/agent.sock"
 
 # This field sets up the KBC that attestation agent uses
 # This is replaced in the makefile steps so do not set it manually
-aa_kbc_params = ""
+aa_kbc_params = "cc_kbc::null"
```

Comment:
What's the reason for setting this to `cc_kbc::null` here?

Reply:
ah, that wasn't intentional. happens at build-time I think.